### PR TITLE
Update upgrade 4.4 docs for post GA

### PR DESCRIPTION
**Pull request by abrennan89**
cc @maschmid @mgencur for QE review
LMK if there are any other changes required for upgrade procedures for 1.7.1.

### CODE REVIEW

File: modules/serverless-upgrade-sub-channel.adoc [hunk 1]
```diff
@@ -5,8 +5,7 @@
 [id="serverless-upgrade-sub-channel_{context}"]
 = Upgrading the Subscription Channel
 
-In {ServerlessProductName} version 1.6.0, the only available Subscription Update Channel was `preview-4.3`.
-To upgrade from 1.6.0 to the latest version, you must update the channel to `4.4`.
+To upgrade from to the latest version of {ServerlessProductName} on {product-title} 4.4, you must update the channel to `4.4`.
```
Comment (mgencur) [inline modules/serverless-upgrade-sub-channel.adoc:8]:
```suggestion
To upgrade to the latest version of {ServerlessProductName} on {product-title} 4.4, you must update the channel to `4.4`.
```

**Review by mgencur — COMMENTED** [review]
One nit , otherwise looks ok.

**Comment by mburke5678** [comment]
@abrennan89 LGTM

**Comment by abrennan89** [comment]
@mburke5678 should be good to merge, thanks :slightly_smiling_face:

**Comment by mburke5678** [comment]
@abrennan89 Should this be merged to master also to keep everything consistent?

https://github.com/openshift/openshift-docs/blob/master/modules/serverless-upgrade-sub-channel.adoc

**Comment by abrennan89** [comment]
@mburke5678 I don't think so, since it's different channels for 4.5?

**Pull request merged**